demo/image_demo.py

```diff
@@ -24,7 +24,7 @@ def main():
         '--title', default='result', help='The image identifier.')
     args = parser.parse_args()
 
-    register_all_modules()
+    init_default_scope('mmseg')
```

Comment:
init_model have add `init_default_scope('mmseg')`,  so there is no need to add it